### ISSUE TITLE
private and file private var braces

### ISSUE THREAD

**Issue by sebleclerc** [nicklockwood/SwiftFormat#546]
Hi! Nice job for the tool! I added it to my existing project and feel a lot better knowing I don't have unnecessary `self.` and braces on another line anymore.

Regarding braces, I think I found a bug. Inside a class, if I create variable as `private` or `fileprivate`, it doesn't get formatted to have the brace on the same line. We have K&R description.

For example:
```
class FooBarClass {
  private var fooDescription: String
  {
    return "Foo is fun"
  }

  var barDescription: String
  {
    return "Bar is also fun"
  }
}
```

When running SwiftFormat, the second var gets it's bracket moved:  
`var barDescription: String {` but the private doesn't.

Thanks 😄

**Comment by nicklockwood** [comment]
@sebleclerc thanks for reporting, I'll look into it.

**Comment by sebleclerc** [comment]
I must add more info to this issue. While reviewing the rest of my codebase, I saw some places where the example above won't get formatted at all. Not even the public property. Sometimes, the opening bracket for a class or extension is not moved beside the declaration.

**Comment by nicklockwood** [comment]
@sebleclerc fixed in 0.44.2.
